exporters/otlp/README.md

```diff
@@ -74,10 +87,10 @@ Sets the number of Goroutines to use when processing telemetry.
 Disables client transport security for the exporter's gRPC connection just like [`grpc.WithInsecure()`](https://pkg.go.dev/google.golang.org/grpc#WithInsecure) does.
 By default, client security is required unless `WithInsecure` is used.
 
-### `WithAddress(addr string)`
+### `WithEndpoint(endpoint string)`
```

Comment:
Can we instead remove this and other configuration documentation from the README? Instead of having to track this documentation in two places, we should make sure the Go docs contain this information and just leave it there.